frontend/model/contracts/shared/functions.js

```diff
@@ -84,7 +83,7 @@ export function getProposalDetails (proposal: Object): Object {
     options['member'] = proposalData.memberName
   } else if (proposalType === PROPOSAL_REMOVE_MEMBER) {
     options['memberID'] = proposalData.memberID
-    options['member'] = sbp('state/vuex/getters').userDisplayNameFromID(proposalData.memberID)
+    // options['member'] = sbp('state/vuex/getters').userDisplayNameFromID(proposalData.memberID)
```

Comment:
Could you either replace this with, or add a comment above this line, explaining where this is now taken care of? (i.e. file & function?)

Reply:
The intent was actually to remove this line. I can add a comment, but the reason to remove this is that contracts can't access external information.